docs/porting-example/index.rst

```diff
@@ -0,0 +1,345 @@
+Porting Example
+===============
+
+HPy supports *incrementally* porting an existing C extension from the
+original Python C API to the HPy API and to have the extension compile and
+run at each step along the way.
+
+Here we walk through porting a small C extension that implements a Point type
+with some simple methods (a norm and a dot product). The Point type is minimal,
+but does contain additional C attributes (the x and y values of the point)
+and an attribute (obj) that contains a Python object (that we will need to
+convert from a `PyObject *` to an `HPyField`).
```

Comment:
Is it intentional that you use single back-quotes? Since this is Restrucutred Text, it will afaik end up in italic style. I usually use double back-quotes (i.e. inline literals).
I'm not strictly against it but we should just do it the same everywhere.

Reply:
I think this should be done. I will push a change.